index.js

```diff
@@ -1,6 +1,43 @@
+const yaml = require('js-yaml')
 const mergeArrayByName = require('./lib/mergeArrayByName')
 
 module.exports = (robot, _, Settings = require('./lib/settings')) => {
+  robot.on('installation', async context => {
+    const { github, payload } = context
+    const { action, repositories, installation } = payload
+    const { account } = installation
+    const { login: repositoryOwner } = account
+
+    if (action === 'deleted') {
```

Comment:
This is unnecessary if you use `installation.created` as the [event](https://developer.github.com/v3/activity/events/types/#installationevent ) name. An example of this is shown here with `issues.opened`
https://probot.github.io/docs/hello-world/